colossalai/inference/modeling/models/nopadding_llama.py

```diff
@@ -107,6 +119,7 @@ def llama_model_forward(
             output_tensor=output_tensor,
             norm_output=norm_output,
             sm_scale=sm_scale,
+            use_cuda=use_cuda,
```

Comment:
`use_cuda` is rather confusing to me. Shouldn't it be always running on GPU? Do you mean `use_fused_cuda_kernel`?

Reply:
Done